action/protocol/staking/protocol.go

```diff
@@ -124,7 +131,12 @@ func (p *Protocol) CreateGenesisStates(
 		return nil
 	}
 
-	csm, err := p.createCandidateStateManager(sm)
+	center, err := getOrCreateCandCenter(sm)
```

Comment:
creategenesisstates is called after protocol started, so this should be get instead of create

Reply:
fixed in latest push